lists-service/src/main/java/au/org/ala/listsapi/controller/GraphQLController.java

```diff
@@ -144,7 +146,7 @@ public Page<SpeciesList> lists(
       }
     }
 
-    NativeQueryBuilder builder = NativeQuery.builder().withPageable(PageRequest.of(1, 1));
+    NativeQueryBuilder builder = NativeQuery.builder().withPageable(PageRequest.of(page, size));
```

Comment:
I think this change should be reverted. The ES output used is that of the aggregation, not the records themselves. I expect this PageRequest applies to the records themselves and not the aggregation.

Reply:
agree - was a previous attempt that didn't work